docs/mp/guides/03_config.adoc

```diff
@@ -85,8 +85,10 @@ For example, if you specify a custom server port in `META-INF/microprofile-confi
 then your server will listen on that port.
 
 A main class is also required to start up the server and run the
-application. If you don't use Helidon's built-in main class you can
-define your own:
+application. By default the Quickstart sample project uses the built-in
+Helidon main. In this guide you want to use your own main so you can have
```

Comment:
Minor point. If we say "main class," then using normal, lower-case "main" seems fine. But if we use "main" by itself--as in "uses the built-in Helidon main" or "use your own main," I feel as if it should be `Main` (capitalized and treated as code).

Reply:
Fixed.